aws/k3s/main.tf

```diff
@@ -0,0 +1,67 @@
+provider "aws" {
+  profile = var.aws_profile
+  region = var.region
+}
+
+locals {
+  user_data = <<DATA
+#cloud-config
+
+runcmd:
+ - apt install curl systemd cgroup-bin cgroup-lite libcgroup1 -y
+ - curl -sfL https://get.k3s.io | sh -
+final_message: "Opsero k3s successfully created, after $UPTIME seconds"
+
+DATA
+}
+
+resource "aws_security_group" "k3s" {
```

Comment:
You can do something like this:

```
resource "aws_security_group" "allow_tls" {
  name        = "allow_tls"
  description = "Allow TLS inbound traffic"
  vpc_id      = "${aws_vpc.main.id}"

  ingress {
    # TLS (change to whatever ports you need)
    from_port   = 443
    to_port     = 443
    protocol    = "tcp"
    # Please restrict your ingress to only necessary IPs and ports.
    # Opening to 0.0.0.0/0 can lead to security vulnerabilities.
    cidr_blocks = # add a CIDR block here
  }

  egress {
    from_port       = 0
    to_port         = 0
    protocol        = "-1"
    cidr_blocks     = ["0.0.0.0/0"]
    prefix_list_ids = ["pl-12c4e678"]
  }
}
```


so that you don't have to create individual rules for each of these. Makes it clearer to see what is being changed.